.github/matrix-jvm-tests.json

```diff
@@ -3,20 +3,23 @@
   "java-version": 17,
   "maven_args": "$JVM_TEST_MAVEN_ARGS",
   "maven_opts": "-Xmx2g -XX:MaxMetaspaceSize=1g",
-  "os-name": "ubuntu-latest"
+  "os-name": "ubuntu-latest",
+  "runs-on": "family=m7a.large/image=ubuntu24-full-x64"
```

Comment:
Don't you need to specify the `${{ github.run_id }}`?

Eg.

```yaml
runs-on:
 - runs-on=${{ github.run_id }}
 - family= m7a.large
 - image= ubuntu24-full-x64
```

Reply:
It's done in the workflow.

Reply:
Ah doh, this is the json, thanks